src/main/java/com/magnatic/DemoApplication.java

```diff
@@ -2,11 +2,15 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
 
 @SpringBootApplication
 public class DemoApplication {
 
+	String one = "1";
 	public static void main(String[] args) {
-		SpringApplication.run(DemoApplication.class, args);
+
+		ConfigurableApplicationContext run =
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "run" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.10.48.216:8090/sonar/coding_rules#rule_key=squid%3AS1481)